src/main/java/com/ibm/crypto/plus/provider/XDHPrivateKeyImpl.java

```diff
@@ -43,6 +43,7 @@ final class XDHPrivateKeyImpl extends PKCS8Key implements XECPrivateKey, Seriali
     private Optional<byte[]> scalar;
     private NamedParameterSpec params;
     private CURVE curve;
+    private byte[] k;
```

Comment:
Perhaps we should add a comment that this is the raw key value to make it obvious this is the key value not the der encoded or octet string encoding of this key value?

Reply:
Done.